packages/kbn-securitysolution-exception-list-components/src/translations.ts

```diff
@@ -11,14 +11,14 @@ import { i18n } from '@kbn/i18n';
 export const EMPTY_VIEWER_STATE_EMPTY_TITLE = i18n.translate(
   'exceptionList-components.empty.viewer.state.empty.title',
   {
-    defaultMessage: 'Add exceptions to this rule',
+    defaultMessage: 'Add exceptions to this list',
```

Comment:
Same here if we pass `emptyViewerButtonText` and the `emptyViewerBody` props they will override the default values defined here